events-api/src/modules/tickets/tickets.service.ts

```diff
@@ -0,0 +1,30 @@
+import { EventsRepository } from "../events/events.repository";
+import { ticketsCreateRequestBodyDto } from "./dto/requests/tickets-create-request-body.dto";
+import { TicketModel } from "./tickets.model";
+import { TicketsRepository } from "./tickets.repository";
+
+export class TicketsService {
+  static async createTicket(
+    data: ticketsCreateRequestBodyDto,
+  ): Promise<TicketModel[] | null> {
```

Comment:
Why to return a list for a singular method?

Reply:
fixed